queries.sql

```diff
@@ -12,4 +12,55 @@ SELECT * FROM animals WHERE neutered = true;
 
 SELECT * FROM animals WHERE name != 'Gabumon';
 
-SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;
\ No newline at end of file
+SELECT * FROM animals WHERE weight_kg BETWEEN 10.4 AND 17.3;   // const user = userData ? JSON.parse(userData) : null
+
+
+ALTER TABLE animals ADD COLUMN species VARCHAR(100);
+
+BEGIN;
+UPDATE animals SET species = 'unspecified';
+SELECT * FROM animals;
+ROLLBACK;
+
+BEGIN;
+UPDATE animals SET species = 'digimon' WHERE name LIKE '%mon';
+UPDATE animals SET species = 'pokemon' WHERE species IS NULL;
+SELECT * FROM animals;
+COMMIT;
```

Comment:
👏 Great progress on updating the `species` column in the `animals` table! 👏 However, to align with the project requirements, there's an additional step needed after the commit. Let's address this to ensure completeness:

- [ ] **Verification After Commit:** After committing the changes to the `animals` table, it's crucial to re-verify, just as you did before the commit. Please add the query to confirm that the changes persist in the database after the commit.
![image](https://github.com/Obimbo07/vet-clinic/assets/55840999/c05010ba-58f8-44dc-a2c1-8185047353d9)
- [ ] Additionally, please consider to add the screenshot in the PR description too.
By including this verification step, your implementation will fully meet the project requirements. Keep up the good work! 🚀🔍